<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentOptionsDialog</class>
 <widget class="QDialog" name="DocumentOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>130</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Document options</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="dimensionsLayout">
    <item>
     <widget class="QSpinBox" name="widthSpinner">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>9000</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="timesLabelSpinner">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Ã—</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="heightSpinner">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>9000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonsLayout">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="workAreaSizeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Workarea size:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
